### ISSUE TITLE
tf.summary.histogram options

### ISSUE THREAD

**Issue by aidangomez** [tensorflow/tensorboard#900]
I have a use case that I think it's pretty common: I want to visualize the categorical probabilities output by a softmax in the form of a histogram.

I.e. I have a vector [0.0 0.5 0.3 0.2] and I want each of those values to specify the height of a bin in a histogram. 

Currently there is no way to do this, but I think this is perhaps *the* most standard modus operandi when preparing data for plotting in a histogram.

**Comment by nfelt** [comment]
See e.g. #1866 which suggests adding `tf.summary.timeseries` (though I'm not sure that's the only type of data we could use the histogram visualization for, either).

**Comment by ruler501** [comment]
Going off the suggestion for timeseries I found a way to implement it using the current APIs. You can probable get rid of the `with` line if you want to avoid experimental APIs.

```
import tensorflow as tf
from tensorboard.plugins.histogram.metadata import create_summary_metadata

def log_timeseries(name, values_array, step=None, description=None, display_name=None):
    with tf.summary.experimental.summary_scope(name, "TimeSeriesSummaryOp", [values_array, step]) as (tag, _):
        return tf.summary.write(
            tag=tag,
            tensor = tf.stack(
                [
                    tf.range(0, values_array.shape[0], dtype=tf.float32),
                    tf.range(1, values_array.shape[0] + 1, dtype=tf.float32),
                    values_array
                ],
                axis=1
            ),
            step=step,
            metadata=create_summary_metadata(display_name, description)
        )
```

**Comment by lminer** [comment]
Just echoing that this would be a nice feature

**Comment by lukysummer** [comment]
Any updates for this plugin?

**Comment by nfelt** [comment]
No updates to share at this point, I'm afraid.
